.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: elixir
+elixir:
+  - 1.1.0
+otp_release:
+  - 17.4
+addons:
+  postgresql: "9.4"
+sudo: false # to use faster container based build environment
+env:
+  - ELIXIR_ERL_OPTIONS=""
+  - ELIXIR_ERL_OPTIONS="+T 9"
```

Comment:
Oh, cool, didn't know about this. https://twitter.com/jlouis666/status/574685606703398913